lib/recart-nacelle-plugin.js

```diff
@@ -1,7 +1,34 @@
 const getMetafieldsForCheckout = require('./getMetafieldsForCheckout')
+const { saveCart } = require('./cart')
 
-const initRecartPlugin = async (ctx, inject) => {
+const delay = async milliseconds => {
+  return new Promise(resolve =>
+    setTimeout(() => resolve(true), milliseconds)
+  )
+}
+
+const initRecartPlugin = async (ctx) => {
   console.log('Recart Nacelle plugin started')
+
+  let timoutCounter = 0
```

Comment:
```suggestion
  let timeoutCounter = 0
```

Reply:
`nit:` actually I'd find `retryCount` or `pollCount` a better name, but up to you